src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -99,6 +99,15 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
             {
                 this.meter.CreateObservableCounter($"{metricPrefix}assembly.count", () => (long)AppDomain.CurrentDomain.GetAssemblies().Length, description: "Number of Assemblies Loaded.");
             }
+
+            if (options.IsExceptionCountEnabled)
+            {
+                var exceptionCounter = this.meter.CreateCounter<long>($"{metricPrefix}exception.count", description: "Number of exceptions thrown.");
```

Comment:
description is not indicating if this is including FirstChance exceptions or not.

Reply:
I still think it's not necessary to mention it in the short description after reading through the the description for [FirstChanceException](https://docs.microsoft.com/en-us/dotnet/api/system.appdomain.firstchanceexception?view=net-6.0) and the comparison between first chance vs second chance exception.

1. [FirstChanceException](https://docs.microsoft.com/en-us/dotnet/api/system.appdomain.firstchanceexception?view=net-6.0) indeed is the count of "exceptions thrown", just with two limits: "in managed code" and "before runtime searches for handler":

> Occurs when an exception is thrown in managed code, before the runtime searches the call stack for an exception handler in the application domain.

2. In comparison, second chance exception [happens when the application doesn't handle it](https://stackoverflow.com/questions/4070040/first-chance-vs-second-chance-exception), i.e. the application crashes, then there is no need to "count" it because it'll always be 0 until it got to 1 and crashes immediately later. The user wouldn't have doubts on whether it's first chance or second chance exception if he/she knows about this concept.

If the user does get question on more detailed definition, he/she can refer to the detailed markdown doc, as for any other metrics in the Runtime metrics.

Is my understanding correct? Do you think it's still needed to add first chance in the description field for some reason?